README.md

```diff
@@ -1,2 +1,154 @@
-# holonix
+# Holonix
+
 Holochain app development environment based on Nix.
+
+## Quickstart - Dev shell from default template
+
+hApp developers can use a Nix command to generate a flake that includes Holochain and other required binaries in a dev shell.
+
+```shell
+nix flake init -t github:holochain/holonix
+```
+
+You may be asked to allow substituters to be set to make use of binaries cached on cachix. This is not required to generate the flake, but when invoking the dev shell, Nix can benefit from the cache and download all binaries instead of building them on your machine from scratch.
```

Comment:
```suggestion
You will be asked to allow substituters to be set to make use of binaries cached on cachix. This is not required to generate the flake, but when invoking the dev shell, Nix can benefit from the cache and download all binaries instead of building them on your machine from scratch.
```

Reply:
It's also worth mentioning that this is local to the project and won't affect their global or user Nix configuration. Even if you already have these configured on your system, it asks anyway and configures your project to use this set